### PR TITLE
Consistent transpose and triplet margin loss test

### PR DESCRIPTION
**Pull request by clackhan**
添加transpose和triplet margin loss  consistent测试用例

### CODE REVIEW

File: python/oneflow/test/modules/test_consistent_TripletMarginLoss.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    m = torch.nn.TripletMarginLoss(margin=margin, p=p, swap=swap, reduction=reduction)
+    m.train(random())
+    anchor = random_pytorch_tensor(2, 8, 16).to_consistent(placement, anchor_sbp)
+    pos = random_pytorch_tensor(2, 8, 16).to_consistent(placement, pos_sbp)
+    neg = random_pytorch_tensor(2, 8, 16).to_consistent(placement, neg_sbp)
+    y = m(anchor, pos, neg)
+    return y
+
+
+class TestConsistentTripletMarginLoss(flow.unittest.TestCase):
```
Comment (clackhan) [inline python/oneflow/test/modules/test_consistent_TripletMarginLoss.py:43]: 这个测试由于 for 循环嵌套较深，需要跑 300+ s。

Comment (hjchen2) [reply python/oneflow/test/modules/test_consistent_TripletMarginLoss.py:43]: 😂

**Review by hjchen2 — APPROVED** [review]

**Review by wyg1997 — APPROVED** [review]

File: python/oneflow/test/modules/test_consistent_TripletMarginLoss.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    @globaltest
+    def test_global_triplet_marginloss_with_random_data(test_case):
+        for placement in all_placement():
+            for anchor_sbp in all_sbp(placement, max_dim=2):
+                for pos_sbp in all_sbp(placement, max_dim=2):
+                    for neg_sbp in all_sbp(placement, max_dim=2):
```
Comment (hjchen2) [inline python/oneflow/test/modules/test_consistent_TripletMarginLoss.py:49]: 这里sbp设置成一样的吧，节约测试时间，其他case半自动推导的单测可以覆盖

File: python/oneflow/test/modules/test_consistent_transpose.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                    arg[0](test_case, placement, sbp)
+
+    @globaltest
+    def test_global_transpose_flow_with_random_data(test_case):
+        for placement in all_placement():
+            for sbp in all_sbp(placement, max_dim=4):
+                _test_global_transpose_flow_with_random_data(test_case, placement, sbp)
+
+    @globaltest
+    def test_global_transpose_with_0_size_data(test_case, valid_split_axis=[0, 1, 3]):
```
Comment (hjchen2) [inline python/oneflow/test/modules/test_consistent_transpose.py:121]: valid_split_axis没用到

Comment (clackhan) [reply python/oneflow/test/modules/test_consistent_transpose.py]: > valid_split_axis没用到

手误，写错地方了，已修改

File: python/oneflow/test/modules/test_consistent_TripletMarginLoss.py (hunk 2 above)
Comment (clackhan) [reply python/oneflow/test/modules/test_consistent_TripletMarginLoss.py]: > 这里sbp设置成一样的吧，节约测试时间，其他case半自动推导的单测可以覆盖

已修改

**Review by hjchen2 — APPROVED** [review]
